pagopa/pn-frontend-e2e
# Test 10395

## Short description

Check download in a cancelled notification

## List of changes proposed in this pull request

- implemented a new test:
   - searching for a specific cancelled notification and entering its detail
   - checks for the absence of the pay button
   - checks if the f24 and the avviso PagoPa are not downloadable
- created some steps for the checks written above

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@checkDownloadFileNotificaAnnullata"` and verify if the test works fine.

Run test on AWS: https://eu-south-1.console.aws.amazon.com/codesuite/codebuild/151559006927/projects/pn-RunFeE2eTestsCodeBuildProject/build/pn-RunFeE2eTestsCodeBuildProject%3Ac476d195-ddd1-4f33-aff5-aab4629fc88f?region=eu-south-1
